nservicebus/messaging/publish-subscribe/index.md

```diff
@@ -34,6 +34,72 @@ When requested by applicative logic to publish a message, the NServiceBus infras
 
 Since one-way messaging is used to dispatch physical messages, even if one of the subscriber endpoints is offline or otherwise unavailable, this does not cause the publishing thread to block. The message is stored in the sending machine's outgoing queue (for a configurable period of time), while the messaging infrastructure attempts to deliver the message to its destination.
 
+## What is auto subscribe
+
+The default mode auto subscribe. When a subscriber endpoint is started it determines to which events it needs to subscribe. It then sends sends subscribtion messages to the dependent publishers. The publishers update their subscription data.
+
+Each time the subscriber is restarted it sends these subscription messages. When you monitor the queing infrastructure you see these subscription messages flowing.
+
+## What happens when a subscriber stops
+
+The default mode is auto subscribe. The subscriber endpoint will never unsubscribe. When the subscriber endpoint stops, it is still registered at the publisher to receive events. The publisher still sends events to the queue of the stopped subscriber. When the subscriber is started it will consume the messages from its queue. The subscriber will never loose an event.
+
+## Can I manually subscribe/unsubscribe
+
+If auto subscribe is unwanted then this can be disabled during initialization. How this is done varies per NServiceBus version. You can explicitly subscribe and unsubscribe via the bus instance. See [How to Publish/Subscribe to a Message](how-to-pub-sub.md) for an example.
```

Comment:
> How this is done varies per NServiceBus version.

remove this. you are writing content that is specific to pub sub and is irrelevant to this page